client/cl_boss.lua

```diff
@@ -133,7 +135,7 @@ end)
 RegisterNetEvent('qb-bossmenu:client:ManageEmployee', function(data)
     local EmployeeMenu = {
         {
-            header = Lang:t('body.mngpl') .. data.player.name .. ' - ' .. string.upper(PlayerJob.label),
+            header = Lang:t('body.mngpl') .. data.player.name .. ' - ' .. string.upper(PlayerJob.label) .. ' (' .. (not not data.player.onduty and 'on duty' or 'off duty') .. ')',
```

Comment:
`PlayerData.job.onduty` is checked at each player login for a nil value, if `onduty` is nil it is set to the default duty value which makes `not not` unnecessary though if `onduty` wasn't checked, this would be a good workaround.